rust/examples/swerve.rs

```diff
@@ -41,10 +41,11 @@ fn main() {
     path.set_bumpers(1.3, 1.3);
     path.pose_wpt(0, 0.0, 0.0, 0.0);
     path.pose_wpt(1, 1.0, 0.0, 0.0);
-    path.wpt_linear_velocity_polar(0, 0.0, 0.0);
-    path.wpt_linear_velocity_polar(1, 0.0, 0.0);
-    path.wpt_angular_velocity(0, 0.0);
-    path.wpt_angular_velocity(1, 0.0);
-    path.sgmt_circle_obstacle(0, 1, 0.5, 0.0, 0.2);
+    // path.wpt_linear_velocity_polar(0, 0.0, 0.0);
+    // path.wpt_linear_velocity_polar(1, 0.0, 0.0);
+    // path.wpt_angular_velocity(0, 0.0);
+    // path.wpt_angular_velocity(1, 0.0);
+    // path.sgmt_circle_obstacle(0, 1, 0.5, 0.1, 0.2);
```

Comment:
Commented out code should be removed. The example segfaulted though, so maybe these lines need to be uncommented instead.

Reply:
the example worked on my machine, the segfault only happened when i tried to run it with the wrong flags